src/config/network/wpa_supplicant.md

```diff
@@ -1 +1,103 @@
-# wicd
+# wpa\_supplicant
+
+The `wpa_supplicant` package is installed by default on the base system
+and includes utilities to configure wireless interfaces and, more
+specifically, to handle wireless security protocols.
+
+[wpa\_supplicant(8)](https://man.voidlinux.org/wpa_supplicant.8) is a
+daemon that manages wireless interfaces based on a
+[wpa\_supplicant.conf(5)](https://man.voidlinux.org/wpa_supplicant.conf.5)
+configuration files.
+[wpa\_passphrase(8)](https://man.voidlinux.org/wpa_passphrase.8) and
+[wpa\_cli(8)](https://man.voidlinux.org/wpa_cli.8) are frontends to
+manage the configuration files and the running service.
+
+## Configuring wpa\_supplicant
+
+First, you must create a configuration file for your interface:
+
+    $ cp /etc/wpa_supplicant/wpa_supplicant.conf /etc/wpa_supplicant/wpa_supplicant-<device>.conf
+
+### WEP Configuration
+
+For WEP configuratoin, you can just add the following lines to your
+device's `wpa-supplicant.conf` :
+
+    # Example default configuration file for wpa_supplicant.conf(5).
```

Comment:
minor point, but this config file could probably be ordered the same as in the package: https://github.com/void-linux/void-packages/blob/master/srcpkgs/wpa_supplicant/files/wpa_supplicant.conf

Reply:
I didn't double check that. These lines were from the wiki. Will do